packages/protocol/test/governance/validators.ts

```diff
@@ -1262,13 +1269,14 @@ contract('Validators', (accounts: string[]) => {
     const validator5 = accounts[7]
     const validator6 = accounts[8]
     const validator7 = accounts[9]
+    const aHash = '0xa5b9d60f32436310afebcfda832817a68921beb782fabf7915cc0460b443116a'
+    const bHash = '0xa832817a68921b10afebcfd0460b443116aeb782fabf7915cca5b9d60f324363'
```

Comment:
Naming nit: So I think in these tests, if you see `aThing`, that usually is literally meant to be read as "a thing", not "thing A". I'd suggest either `hash1`, `hash2`, or `aHash`, `anotherHash`.